pyproject.toml

```
@@ -227,13 +227,16 @@ warn_unused_ignores = true
 
 [tool.numpydoc_validation]
 checks = [
+  "all",
+  # all but the following:
```

Inline comments are considered better for compatibility with the formatter in my mind.
```suggestion
  "all", # all but the following:
```

---

I noticed this comment has a colon, but it was at the bottom of the list. It seems to make more sense for `all` (and its comment) to be the first entry. But, it's not a necessary change, we can revert it.

Making the comment _not_ inline in this case breaks the sorting into two (above and below are sorted independently). This was considered a bug here https://github.com/pyvista/pyvista/pull/6781#discussion_r1838729385, but for this PR it's a feature. If we keep the comment inline, we'll need a blank comment after if we want `all` to be first.

```suggestion
  "all", # all but the following:
  #
```